robotidy/transformers/SplitTooLongLine.py

```diff
@@ -2,18 +2,41 @@
     ModelTransformer,
     Token
 )
+from robotidy.decorators import check_start_end_line
 
 
 EOL = Token(Token.EOL)
 CONTINUATION = Token(Token.CONTINUATION)
 
 
 class SplitTooLongLine(ModelTransformer):
+    """
+    Split too long lines.
+    If any line in keyword call exceed given limit (configurable using ``line_length``. Default is 120) it will be
```

Comment:
Only in keyword call or anywhere in the code?

Reply:
Only for keyword call. We need to add support for other places in the future (like settings, headers, variables)

Reply:
Small grammar / styling nit, just to ensure that my GitHub bio is up to date ;)

```suggestion
    If any line in keyword call exceeds given length limit (configurable using ``line_length``, 120 by default) it will be
```